pytextnow/TNAPI.py

```diff
@@ -108,24 +108,26 @@ def get_initial_csrf_token(self):
             token_end = resp.find('"', token_start)
             csrf_token = resp[token_start:token_end]
 
-            self.headers['x-csrf-token'] = csrf_token
+            return csrf_token
+        else:
+            raise FailedRequest(req.status_code)
 
     def auth_reset(self, cookie=None):
-        with open(self._user_sid_file, "r") as user_SIDS_file:
-            user_SIDS = json.loads(user_SIDS_file.read())
+        with open(self._user_cookies_file, "r") as user_cookie_file:
+            user_cookies = json.loads(user_cookie_file.read())
 
-        if self.username in user_SIDS.keys():
-            del user_SIDS[self.username]
+        if self.username in user_cookies.keys():
+            del user_cookies[self.username]
 
-            with open(self._user_sid_file, "w") as user_SIDS_file:
-                user_SIDS_file.write(json.dumps(user_SIDS))
+            with open(self._user_cookies_file, "w") as user_cookies_file:
+                user_cookies_file.write(json.dumps(user_cookies))
 
             self.__init__(self.username, cookie)
         else:
             if cookie:
-                user_SIDS[self.username] = cookie
-                with open(self._user_sid_file, "w") as user_SIDS_file:
-                    user_SIDS_file.write(json.dumps(user_SIDS))
+                user_cookies[self.username] = cookie

```

Comment:
`cookie` in this case will have to be a dict with 'sid' and 'csrf' keys. Do you want to change the method to accept an sid_cookie and a csrf_cookie separately like I did with __init__? Or will you just expect the user to pass in a dict?

Reply:
Oh that was a mistake lol thnx for pointing it out

Reply:
Alright I pushed a new commit